other/perun-slave-metacentrum/rpm.dependencies

```diff
@@ -1 +1 @@
-perun-slave-process-afs, perun-slave-process-afs-group, perun-slave-process-apache-basic-auth, perun-slave-process-apache-ssl, perun-slave-process-fs-home, perun-slave-process-fs-project, perun-slave-process-fs-quotas, perun-slave-process-fs-scratch, perun-slave-process-fs-scratch-local, perun-slave-process-group, perun-slave-process-group-nfs4, perun-slave-process-k5login, perun-slave-process-k5login-generic, perun-slave-process-k5login-root, perun-slave-process-mailaliases, perun-slave-process-mailaliases-generic, perun-slave-process-mailman-meta, perun-slave-process-passwd, perun-slave-process-passwd-nfs4, perun-slave-process-passwd-scp, perun-slave-process-pbsmon-json, perun-slave-process-pbsmon-users, perun-slave-process-pbs-phys-cluster, perun-slave-process-pbs-pre, perun-slave-process-pbs-publication-fairshare, perun-slave-process-pkinit, perun-slave-process-sshkeys
+perun-slave-process-afs, perun-slave-process-afs-group, perun-slave-process-apache-basic-auth, perun-slave-process-apache-shibboleth, perun-slave-process-apache-ssl, perun-slave-process-fs-home, perun-slave-process-fs-project, perun-slave-process-fs-quotas, perun-slave-process-fs-scratch, perun-slave-process-fs-scratch-local, perun-slave-process-group, perun-slave-process-group-nfs4, perun-slave-process-k5login, perun-slave-process-k5login-generic, perun-slave-process-k5login-root, perun-slave-process-mailaliases, perun-slave-process-mailaliases-generic, perun-slave-process-mailman-meta, perun-slave-process-passwd, perun-slave-process-passwd-nfs4, perun-slave-process-passwd-scp, perun-slave-process-pbsmon-json, perun-slave-process-pbsmon-users, perun-slave-process-pbs-phys-cluster, perun-slave-process-pbs-pre, perun-slave-process-pbs-publication-fairshare, perun-slave-process-pkinit, perun-slave-process-sshkeys
```

Comment:
I'm not sure, that this service is required by Metacentrum infrastructure, hence it shouldn't be as dependency for this meta-package.